config/v1/types_oauth.go

```diff
@@ -139,50 +141,16 @@ const (
 	// that user name already exists, the identity is mapped to the existing user, adding to any
 	// existing identity mappings for the user.
 	MappingMethodAdd MappingMethodType = "add"
-
-	// MappingMethodGenerate provisions a user with the identity’s preferred user name. If a user
-	// with the preferred user name is already mapped to an existing identity, a unique user name is
-	// generated, e.g. myuser2. This method should not be used in combination with external
-	// processes that require exact matches between openshift user names and the idp user name
-	// such as LDAP group sync.
-	MappingMethodGenerate MappingMethodType = "generate"
-)
-
-// GrantHandlerType are the valid strategies for handling grant requests
-type GrantHandlerType string
-
-const (
-	// GrantHandlerAuto auto-approves client authorization grant requests
-	GrantHandlerAuto GrantHandlerType = "auto"
-	// GrantHandlerPrompt prompts the user to approve new client authorization grant requests
-	GrantHandlerPrompt GrantHandlerType = "prompt"
-	// GrantHandlerDeny auto-denies client authorization grant requests
-	GrantHandlerDeny GrantHandlerType = "deny"
 )
 
 type IdentityProviderType string
 
 const (
-	// IdentityProviderTypeBasicAuth provides identities for users authenticating with HTTP Basic Auth
-	IdentityProviderTypeBasicAuth IdentityProviderType = "BasicAuth"
-
 	// IdentityProviderTypeAllowAll provides identities for all users authenticating using non-empty passwords
 	IdentityProviderTypeAllowAll IdentityProviderType = "AllowAll"
 
-	// IdentityProviderTypeDenyAll provides no identities for users
-	IdentityProviderTypeDenyAll IdentityProviderType = "DenyAll"
```

Comment:
where did this go?

Reply:
Implied by no IDPs as it does not perform short circuit deny.